telco-ran/configuration/machineconfigs/misc/75-rhcos4-etc_audit_rules.d_75-audit-sysadmin-actions.rules-combo.yaml

```diff
@@ -0,0 +1,23 @@
+# Combined from the following remediations for /etc/audit/rules.d/75-audit-sysadmin-actions.rules (all roles):
```

Comment:
the file name here is awful, cannot understand what this file is about from that
if it's just for sudo, call it maybe audit-sudoers ?